core/bin/external_node/src/main.rs

```diff
@@ -539,36 +527,9 @@ async fn init_tasks(
     stop_receiver: watch::Receiver<bool>,
     components: &HashSet<Component>,
 ) -> anyhow::Result<()> {
-    let release_manifest: serde_json::Value = serde_json::from_str(RELEASE_MANIFEST)
-        .context("releuse manifest is a valid json document")?;
-    let release_manifest_version = release_manifest["core"].as_str().context(
-        "a release-please manifest with \"core\" version field was specified at build time",
-    )?;
-
-    let version = semver::Version::parse(release_manifest_version)
-        .context("version in manifest is a correct semver format")?;
-    let pool = connection_pool.clone();
-    let mut stop_receiver_for_task = stop_receiver.clone();
-    task_handles.push(tokio::spawn(async move {
-        while !*stop_receiver_for_task.borrow_and_update() {
-            let protocol_version = pool
-                .connection()
-                .await?
-                .protocol_versions_dal()
-                .last_used_version_id()
-                .await
-                .map(|version| version as u16);
-
-            EN_METRICS.version[&(version.to_string(), protocol_version)].set(1);
```

Comment:
Were the old metrics used anywhere? Probably obvious, but this PR has changed their naming and split off protocol version into a gauge (since it can change during the program runtime).